rotkehlchen/tests/exchanges/test_kucoin.py

```diff
@@ -0,0 +1,727 @@
+    should be skipped.
+    """
+    expected_trades = [
+        Trade(
+            timestamp=Timestamp(1612556765),
+            location=Location.KUCOIN,
+            pair=TradePair('ETH_BTC'),
+            trade_type=TradeType.BUY,
+            amount=AssetAmount(FVal('0.02934995')),
+            rate=Price(FVal('0.046058')),
+            fee=Fee(FVal('9.4625999797E-7')),
+            fee_currency=Asset('BTC'),
+            link='601da9ddf73c300006194ec6',
+            notes='',
+        ),
+        Trade(
+            timestamp=Timestamp(1612556765),
+            location=Location.KUCOIN,
+            pair=TradePair('ETH_BTC'),
+            trade_type=TradeType.BUY,
+            amount=AssetAmount(FVal('0.02')),
+            rate=Price(FVal('0.04561')),
+            fee=Fee(FVal('6.3854E-7')),
+            fee_currency=Asset('BTC'),
+            link='601da9ddf73c300006194ec5',
+            notes='',
+        ),
+        Trade(
+            timestamp=Timestamp(1612556765),
+            location=Location.KUCOIN,
+            pair=TradePair('ETH_BTC'),
+            trade_type=TradeType.BUY,
+            amount=AssetAmount(FVal('0.06')),
+            rate=Price(FVal('0.0456')),
+            fee=Fee(FVal('0.0000019152')),
+            fee_currency=Asset('BTC'),
+            link='601da9ddf73c300006194ec4',
+            notes='',
+        ),
+        Trade(
+            timestamp=Timestamp(1612556693),
+            location=Location.KUCOIN,
+            pair=TradePair('BTC_USDT'),
+            trade_type=TradeType.SELL,
+            amount=AssetAmount(FVal('0.0013')),
+            rate=Price(FVal('37624.4')),
+            fee=Fee(FVal('0.034238204')),
+            fee_currency=Asset('USDT'),
+            link='601da995e0ee8b00063a075c',
+            notes='',
+        ),
+    ]
+    trades = sandbox_kuckoin.query_online_trade_history(
+        start_ts=Timestamp(1612556693),
+        end_ts=Timestamp(1612556765),
+    )
+    assert trades == expected_trades
+
+
+@pytest.mark.parametrize('should_mock_current_price_queries', [True])
+def test_query_asset_movements_sandbox(
+        sandbox_kuckoin,
+        inquirer,  # pylint: disable=unused-argument
+):
+    """Unfortunately the sandbox environment does not support deposits and
+    withdrawals, therefore they must be mocked.
+
+    The sandbox account has 6 movements. Below a list of the movements and their
+    timestamps in ascending mode.
+    - movement 1 - deposit: 1612556651 -> skipped
+    - movement 2 - deposit: 1612556693
+    - movement 3 - withdraw: 1612556765 -> skipped, inner withdraw
+    - movement 4 - deposit: 1612556765 -> skipped, inner deposit
+    - movement 5 - withdraw: 1612556765
+    - movement 6 - withdraw: 1612556794 -> skipped
+
+    By requesting trades from 1612556693 to 1612556765, the first and last
+    movement should be skipped, but also the two inner movements.
+    """
+    deposits_response = (
+        """
+        {
+            "code":"200000",
```

Comment:
how did you figure out correctness of mock data here?

Reply:
Took it from their documentation
Deposits list: https://docs.kucoin.com/#get-deposit-list
Withdrawals list: https://docs.kucoin.com/#get-withdrawals-list